CONTRIBUTING.md

```diff
@@ -88,18 +89,19 @@ github.com/jaegertracing/jaeger
   mkdocs.yml                - MkDocs builds the documentation in docs/
 ```
 
-  * Note 1: `pkg` is a collection of utility packages used by the Jaeger components
-    without being specific to its internals. Utility packages are kept separate from
-    the Jaeger core codebase to keep it as small and concise as possible. If some
-    utilities grow larger and their APIs stabilize, they may be moved to their own
-    repository, to facilitate re-use by other projects.
+- Note 1: `pkg` is a collection of utility packages used by the Jaeger components
```

Comment:
nit: limit your PR to only changes which are required. `*` is a valid char for list and removing spaces does not change the formatting either.